drivers/SmartThings/zwave-switch/fingerprints.yml

```diff
@@ -784,11 +784,11 @@ zwaveManufacturer:
     productType: 0x1801
     productId: 0x1000
     deviceProfileName: metering-switch
-  - id: 0086/004E
-    deviceLabel: Aeotec Switch
+  - id: "Aeotec/Heavy Duty"
```

Comment:
I may be remembering this wrong but I thought we couldn't update the ID of an existing fingerprint. @greens ?